### DISCUSSION TITLE
我的家庭观影之路

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#110] [Announcements]
https://www.xheldon.com/life/the-way-to-watching-tv.html 

 为家人实现观影自由，也是一种幸福。

**Comment by hongzzz** [comment]
跟我现在的方案十分像哈哈哈

**Comment by Xheldon** [comment]
> @hongzzz
> 跟我现在的方案十分像哈哈哈

缘分~

**Comment by bestrooney** [comment]
太好了，跟我的一模一样，整个方案也是我想要的，我比博主多了个看球赛😄

**Comment by sunnicholas** [comment]
优点复杂

**Reply by Xheldon** [reply]
如果你有一个自己的房子和大电视，而且还不想花太多钱（腾讯、优酷、爱奇艺、Netflix、迪斯尼等），甚至花钱买罪受（会员额外付费、会员专属广告、阉割、众所周知原因下架优秀作品），折腾这些是迟早的事情😏
